go.mod

```diff
@@ -1,46 +1,46 @@
 module github.com/keikoproj/active-monitor
 
-go 1.19
+go 1.21
 
 require (
 	github.com/argoproj/argo-workflows/v3 v3.4.11
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-logr/logr v1.2.4
 	github.com/keikoproj/inverse-exp-backoff v0.0.3
 	github.com/mitchellh/mapstructure v1.5.0
-	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/ginkgo/v2 v2.13.0
 	github.com/onsi/gomega v1.27.10
 	github.com/prometheus/client_golang v1.16.0
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/sirupsen/logrus v1.9.3
 	github.com/stretchr/testify v1.8.4
-	golang.org/x/net v0.15.0
+	golang.org/x/net v0.17.0
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.25.14
-	k8s.io/apimachinery v0.25.14
-	k8s.io/client-go v0.25.14
-	sigs.k8s.io/controller-runtime v0.13.1
+	k8s.io/api v0.28.3
+	k8s.io/apimachinery v0.28.3
+	k8s.io/client-go v0.28.3
```

Comment:
@tekenstam should we upgrade to `0.26.x` instead? to be compatible with `k8s 1.26`